.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for your pull request. Please provide a description above and review
+the requirements below.
+
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] `npm test` passes
+- [ ] tests are included
+- [ ] documentation is changed or added
+- [ ] commit message follows commit guidelines
```

Comment:
we should link to the commit guidelines here, if we don't have any, we can always link to what's in nodejs/node